packages.html

```diff
@@ -0,0 +1,334 @@
+
+</p>  
+              
+      </section>
+		
+      <section class="box content" id="core">
+          
+              <h4>Core</h4>
+<p>
+A package with the designation "Core" has several key features:</p>
+
+<div id="plain">
+<ul>
+  <li>Critically important to a wide and general audience</li>
+  <li>Published, peer-reviewed documentation on the package function</li>
+  <li>Long-term support addressed publicly (could be stated in README or elsewhere)</li>
+  <li>Rapid response to bug fixes</li>
+  <li>Rapid response to R version upgrades</li>
+  <li>Rapid response to R package dependency changes</li>
+  <li>Customer support and troubleshooting</li>
+  <li>An active USGS employee is designated as maintainer. Development and maintenance of core packages should include a team of developers. Maintainers coordinate and lead the development team, but should not be the sole developer.</li>
+  <li>The package should be well tested, and the package should be tested on a continuous integration platform regularly.</li>
+  <li>All external functions should have clear documentation and working examples.</li>
+</ul>
+</div>
+
+<p>
+A user of core packages can expect prompt feedback from bug reports and general-use questions. Users can expect smooth transitions of maintainers since the package is well-tested, well-documented, and multiple developers have collaborated on the project. Enhancements should be carefully considered, "scope-creep" should be actively avoided (that is, careful thought to add new features). The general scope of the package should not change over time.  </p>            
+              
+<h3>Example Core Packages</h3>
+  
+  <div id="wrapper" class="grid clearfix">
+      <div id="repoContainer">
+         <a href="https://github.com/USGS-R/dataRetrieval" target="_blank">
+            <div class="repos coreColor">
+              <h2>dataRetrieval</h2>
+              <p class="repoDescription">This R package for downloading and importing USGS and EPA water data.</p>
+              <p class="underDevelopment">Core</p>
+            </div>
+          </a>
+          <a href="https://github.com/USGS-R/geoknife" target="_blank">
+            <div class="repos coreColor">
+            	<h2>geoknife</h2>
+                <p class="repoDescription">Tools for geo-web processing of gridded data. geoknife slices up data according to overlap with irregular features.</p>
+                <p class="underDevelopment">Core</p>
+            </div>
+        	</a>       
+             
+     </div>
+  </div>
+      </section>
+      
+      <section class="box content" id="research">
+          
+              <h4>Research</h4>
+              
+              <p>Research packages have three distinct phases: Active, Archive, and Exploratory.</p>
```

Comment:
put exploratory first? i know it's less important than the others, but as a phase it surely comes first if at all...